src/unnamed-engine/include/core/log/Logger.hpp

```diff
@@ -0,0 +1,115 @@
+//
+//  ScriptingManager.hpp
+//  embedding_test
+//
+//  Created by Leonidas Neftali Gonzalez Campos on 25/12/23.
+//
+
+#pragma once
+
+#include <spdlog/spdlog.h>
+
+enum class LogLevel : std::uint32_t
```

Comment:
Precede this Enum with the E prefix as `ELogLevel`

Reply:
Yep, going to do that once #27 is merged.